### ISSUE TITLE
Error in services.yaml for Symfony 5.4.0

### ISSUE THREAD

**Issue by smoelker** [pixelfederation/swoole-bundle#46]
**Describe the bug**
I'm a first time user of Swoole & this bundle. I followed the installation instructions and get the following error when I run `bin/console swoole:server:run`

```
In AbstractRecursivePass.php line 119:
                                                                                                                                                
  Invalid service "K911\Swoole\Server\Api\ApiServerClient": function "K911\Swoole\Server\Api\ApiServerClientFactory:newClient" does not exist.
```

Everything works fine and life is good again when I change `services.yaml` from
```yaml
K911\Swoole\Server\Api\ApiServerClient:
    factory: 'K911\Swoole\Server\Api\ApiServerClientFactory:newClient'
```

to

```yaml
K911\Swoole\Server\Api\ApiServerClient:
    factory: ['K911\Swoole\Server\Api\ApiServerClientFactory', 'newClient']
```

This error does not occur on Symfony 5.4.11 so I'll try to upgrade.

**Steps To Reproduce**
Follow installation instructions, lock symfony on version 5.4.0 and start server.

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04
 - PHP Version: 8.0.21 and 8.1.8
 - Swoole Version: 4.11.1
 - Symfony Version: 5.4.0
 - Running in docker: No

**Comment by Rastusik** [comment]
Hi @smoelker ! Thank you for the issue report and prompt fix, really appreciated! 
I think it would be worth investigating what the root cause of the problem is, because our CI pipelines run just fine without your change with PHP 8 and 8.1 and with latest Symfony, which is kind of weird and I'm unable to reproduce your issue. Even in our company projects, there isn't anything to be found. Do you have any special setting somewhere?

**Comment by smoelker** [comment]
I can reproduce the issue by pinning Symfony to version 5.4.0. The issue seems to be fixed as of Symfony version 5.4.3. I can't find anything in the release notes of the versions in between that would explain the change in behavior. 

My guess would be that CI pipeline does not test against version 5.4.0 and therefore passes.

Script to reproduce the error:

```bash
#!/bin/bash

# Initialize Symfony webapp v5.4.*
symfony new reproduce --version="5.4.*" --webapp
cd reproduce

# Pin Symfony to 5.4.0
sed -i 's/5\.4\.\*/5\.4\.0/' composer.json
composer update --with-all-dependencies

# Add SwooleBundle to bundles.php
sed -i -z 's/\n]\;/\n\tK911\\Swoole\\Bridge\\Symfony\\Bundle\\SwooleBundle::class => ['"'"'all'"'"' => true],\n]\;/' config/bundles.php

# Install SwooleBundle
composer require pixelfederation/swoole-bundle swoole/ide-helper:@dev

bin/console swoole:server:run

```

I'll revisit the pull request to pass ci checks tomorrow I hope.

**Comment by Rastusik** [comment]
@smoelker thank you very much for the clarification. I'll merge the PR after it is in order according to the review.
